client/src/components/Tour/TourList.vue

```diff
@@ -0,0 +1,77 @@
+<template>
+    <div class="ui-thumbnails">
+        <h2>Galaxy Tours</h2>
+        <p>
+            This page presents a list of interactive tours available on this Galaxy server. Select any tour to get
+            started (and remember, you can click 'End Tour' at any time).
+        </p>
+        <h4>Tours</h4>
+        <div v-if="error" class="alert alert-danger">{{ error }}</div>
+        <div v-else>
+            <DelayedInput class="mb-3" :query="search" :placeholder="searchTours" :delay="100" @change="onSearch" />
```

Comment:
Since this is purely clientside the delay seems unnecessary; you could bind the value of the input to `search` directly and this would feel snappier.

Reply:
@dannon good callout - delay removed, but input box kept for consistency with `WorkflowList.vue`